sphinx/api/configuration/site-and-user.rst

```diff
@@ -14,10 +14,18 @@ site configuration at ``$ROSE_HOME/etc/rose.conf`` where
 search for the user configuration at ``$HOME/.metomi/rose.conf`` where
 ``$HOME`` is the home directory of the current user.
 
-You can also override many internal constants of the ``rose config edit`` and
+.. note::
```

Comment:
This note is no longer applicable, the ``rose.conf`` settings section having being added in now.